src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/reported/WitnessesController.kt

```diff
@@ -13,7 +13,7 @@ import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.controllers.draft.WitnessesRequest
 import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.services.reported.WitnessesService
 
-@PreAuthorize("hasAuthority('SCOPE_write')")
+@PreAuthorize("hasRole('ADJUDICATIONS_REVIEWER') and hasAuthority('SCOPE_write')")
```

Comment:
this is not ALO specific

Reply:
indeed - see https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467#discussion_r1286774976